packages/e2e-tests/test-applications/generic-ts3.8/index.ts

```diff
@@ -1,13 +1,2 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 // we need to import the SDK to ensure tsc check the types
-import * as _SentryBrowser from '@sentry/browser';
```

Comment:
Do we need this file?

Reply:
I think we do, it's to test TS 3.8 types iiuc

Reply:
Fixed with 51830a5c2f87df43d39cf947a4f63c6980f67aeb